py/desispec/workflow/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+desispec.workflow
+===========
+
+Tools for workflow management and running the pipeline
+"""
+from __future__ import absolute_import
```

Comment:
FWIW, this `from __future__ ...` stuff was for the python 2.x -> 3.x transition, and we no longer need it.

Reply:
I assumed as much, but it still existed in another `__init__.py `within desisepc, so I included it out of an abundance of caution.

Reply:
I'll leave this in and let a future refactor that cleans up all of these __future__ items clean it up